packages/fx-core/templates/plugins/resource/function/bicep/function.template.bicep

```diff
@@ -0,0 +1,125 @@
+param functionServerfarmsName string
```

Comment:
Please also add tests case for generating function bicep files. Or test coverage will reduce.